tor_core/initialize.py

```diff
@@ -230,7 +230,7 @@ def build_bot(name, log_name='transcribersofreddit.log', require_redis=True):
         # I'm sorry
```

Comment:
I want to throw an exception there, but lambdas don't allow statements, only expressions. Turns out that `raise NotImplementedError` is a statement, so you have to use an expression (empty generator) and then throw an exception when the empty generator finishes running. It's hacky and kinda gross, but it's really clear what happens when it runs, though 😆